### PR TITLE
Update user import

### PR DESCRIPTION
**Pull request by sambuddhabasu**

### CODE REVIEW

**Comment by mitechie** [comment]
:+1: Thanks for the update.

File: bookie/lib/importer.py [hunk 1]
```diff
@@ -181,9 +182,9 @@ def process(self):
 
             link = tag.a
 
-            # Skip any bookmarks with an attribute of PRIVATE.
-            if link.has_key('PRIVATE'):  # noqa
-                continue
+            is_private = False
+            if link.has_key('private'):  # noqa
```
Comment (craigmaloney) [inline bookie/lib/importer.py:186]: Wondering if we should do a search in the keys to cover "private", "is_private" or any other version of this.

That said, maybe that's more trouble than its worth.

**Comment by craigmaloney** [comment]
:+1:

File: bookie/lib/importer.py (hunk 1 above)
Comment (sambuddhabasu) [inline bookie/lib/importer.py:186]: For DelImporter, the private bookmarks are marked with a 'PRIVATE="1"', so we are searching here with key 'private'.

**Comment by sambuddhabasu** [comment]
:shipit:

**Pull request merged**